package-lock.json

```diff
@@ -1,8 +1,6013 @@
 {
   "name": "github-action-merge-dependabot",
   "version": "2.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
please use npm@6 to avoid changing the version of the lockfile

Reply:
No problem, I've reverted it back to v1 👍🏻